WordPress/Classes/ViewRelated/Comments/EditCommentViewController.m

```diff
@@ -85,11 +87,9 @@ - (void)viewWillDisappear:(BOOL)animated
     [super viewWillDisappear:animated];
     
     // FIX FIX:
-    // iOS 8 is resigning first responder when the presentedViewController is effectively removed from screen.
+    // iOS +8 is resigning first responder when the presentedViewController is effectively removed from screen.
```

Comment:
8+ ?

Reply:
The guard i've nuked a few lines below, was checking for iOS +8. So... nuked the guard, updated the comment

Reply:
Sorry I wasn't clear on my comment, I meant that we want to say iOS 8+ instead of iOS +8, if that makes sense.

Reply:
Ahhhh gotcha. Will update, definitely makes sense, thank you!